src/OptionWrapper.js

```diff
@@ -20,6 +20,9 @@
           window.addEventListener('mouseup', listener);
           return cancelEvent(e);
         },
+        'aria-selected': this.props.highlight,
+        id: this.props.item && this.props.item.value,
```

Comment:
what happens if you have options with the same values?

Reply:
It turns out react-selectize can't really handle duplicate value options in a clean way, so from the perspective of this id here it should be a precondition that they're already deduped.  However, I've updated this code so that it's prefixed with the listboxId (to prevent duplicates if there are more than two SimpleSelects being rendered with similar options).